htdocs/lost_password.php

```diff
@@ -56,7 +56,7 @@
 
             // reset the password in the database
             // expire password so user must change it upon login
-            $success = $user->updatePassword($password, '0000-00-00');
+            $success = $user->updatePassword($password, '1999-01-01');
```

Comment:
@ridz1208 I read somewhere that Loris was first created in 1999, so I used it as a symbolic date